docs/index.md

```diff
@@ -85,3 +85,8 @@ The following arguments are supported in the `provider` block:
              It can be sourced from login in to your CloudAMQP account and go to API access or go
              directly to [API Keys](https://customer.cloudamqp.com/apikeys).
              The API key can also be read from the environment variable `CLOUDAMQP_APIKEY`.
+
+* `skip_on_destroy` (Optional) This argument can be set to skip delete behaviour for resources on
+                    `terraform destroy`. Each resource supporting this have a section on its respective page.
+                    The argument can also be read from the environment variable `CLOUDAMQP_SKIP_ON_DESTROY`,
+                    default set to false.
```

Comment:
Maybe this section should also list the resources which support/make use of this?

We can also include that it will speed up `terraform destroy`

Maybe there's a better name for this... "skip on destroy" isn't _really_ true... I mean, the resources will be removed from the state, right? What we skip is the HTTP requests to the API to delete the resources – oh well, I have not good name in my right now, just throwing out some thoughts :)

Reply:
Good points. Heh yes, not the easiest to come up with a good name. Some name that plopped up this morning.
- skip_backend_destroy
- skip_backend_deletion
- skip_backend_removal

Reply:
I think it makes more sense for this option to be per resource instead of global? WDYT?

Reply:
If keeping it as global... maybe it can be named `enable_faster_instance_destroy`?

Reply:
Though about both cases and landed in the global option. To hit all resources that we consider can be skipped in regards to `cloudamqp_instance` in one configuration.

Will update with your name suggestion and, then we can take a another look.